examples/web-server/caddy/docker-compose.yaml

```diff
@@ -0,0 +1,57 @@
+version: "3.9"
```

Comment:
What is the purpose of `~/docker-compose.yaml` vs `~/examples/web-server/caddy/docker-compose.yaml`? 

ie. does it make sense to delete one?

Reply:
`~/docker-compose.yaml` uses the tunnel by default which means that Caddy + a domain name is not required. However, I think we could move the compose to `~/examples/install/docker` or something.

Reply:
👍 re:  I think we could move the compose to ~/examples/install/docker or something in an additional PR.